src/norm/sqlite.nim

```diff
@@ -364,16 +365,44 @@ proc selectOneToMany*[O: Model, M: Model](dbConn; oneEntry: O, relatedEntries: v
   dbConn.select(relatedEntries, sqlCondition, oneEntry.id)
 
 proc selectOneToMany*[O: Model, M: Model](dbConn; oneEntry: O, relatedEntries: var seq[M]) =
-  ## A convenience proc. Fetches all entries of a "many" side from a one-to-many 
-  ## relationship between the model of `oneEntry` and the model of `relatedEntries`.
-  ## The field used to fetch the `relatedEntries` is automatically inferred as long
-  ## as the `relatedEntries` model has only one field pointing to the model of 
-  ## `oneEntry`. Will not compile if `relatedEntries` has multiple fields that 
-  ## point to the model of `oneEntry`. Specify the `foreignKeyFieldName` parameter 
-  ## in such a case.
+  ## A convenience proc. Fetches all entries of a "many" side from the single one-to-many 
```

Comment:
This is only a minor doc change, it felt too inaccurate after I read over it again, so I adjusted the wording a smidge.